src/inverseRobot.js

```diff
@@ -24,7 +24,23 @@
  * @return {object}
  */
 function inverseRobot(robot) {
-  // write code here
+  const newRobot = {};
+  const keys = Object.keys(robot);
+  const values = Object.values(robot);
+
+  if (keys.length === 0) {
+    return {};
+  }
+
+  for (let i = 0; i < keys.length; i++) {
```

Comment:
You can also use for (const key in robot) cycle instead of creating keys and values arrays